File: python/health.py
import os
import httpx

BASE_URL = os.environ.get("NPI_API_BASE_URL", "https://healthproviderapi.com")


def health_check() -> dict:
    response = httpx.get(f"{BASE_URL}/api/health", timeout=10.0)
    response.raise_for_status()
    return response.json()


if __name__ == "__main__":
    data = health_check()
    print("API is healthy:", data)

File: node/health.js
import 'dotenv/config';

const BASE_URL = process.env.NPI_API_BASE_URL || 'https://healthproviderapi.com';

async function healthCheck() {
  const res = await fetch(`${BASE_URL}/api/health`);
  if (!res.ok) throw new Error(`Health check failed: ${res.status}`);
  const data = await res.json();
  console.log('API is healthy:', JSON.stringify(data));
  return data;
}

healthCheck().catch(err => { console.error(err.message); process.exit(1); });
